app/src/main/res/values/styles.xml

```diff
@@ -238,11 +238,24 @@
     <item name="fontFamily">sans-serif-medium</item>
   </style>
 
+  <style name="DefaultButton" parent="Button.Common">
+    <item name="android:textSize">@dimen/default_button_text_size</item>
+  </style>
+
   <!-- This style should not be used in layout/*.xml files.-->
   <style name="TextView.Common1" parent="Widget.AppCompat.TextView">
     <item name="android:layout_width">wrap_content</item>
     <item name="android:layout_height">wrap_content</item>
     <item name="android:textAllCaps">false</item>
     <item name="android:textColor">@color/oppiaPrimaryText</item>
   </style>
+
+  <style name="Button.Common" parent="Widget.AppCompat.Button">
+    <item name="android:layout_width">match_parent</item>
+    <item name="android:layout_height">wrap_content</item>
+    <item name="android:textColor">@color/white</item>
+    <item name="android:background">@drawable/primary_rounded_button</item>
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:minHeight">48dp</item>
+  </style>
```

Comment:
This is not needed. Please revert the changes in this file.

Reply:
Reverted